lib/archethic_web/graphql_schema/resolver.ex

```diff
@@ -33,6 +34,54 @@ defmodule ArchethicWeb.GraphQLSchema.Resolver do
     end
   end
 
+  def get_token(address) do
+    genesis_address =
```

Comment:
You should parallelize into two tasks and await the result to make it more efficient